mergin/client.py

```diff
@@ -623,7 +623,7 @@ def projects_list(
                 break
         return projects
 
-    def project_info(self, project_path, since=None, version=None):
+    def project_info(self, project_path_or_id, since=None, version=None):
```

Comment:
please fix the function docstring for project_path param...

Reply:
done